src/chocolatey.resources/helpers/functions/Install-PSModulePath.ps1

```diff
@@ -0,0 +1,56 @@
+﻿# Copyright 2011 - Present RealDimensions Software, LLC & original authors/contributors from https://github.com/chocolatey/chocolatey
```

Comment:
This will be a little different since it originates here. The `& original...` should be removed.